Dockerfile

```diff
@@ -111,7 +111,18 @@ RUN apt-get update \
         jq \
  && gem cleanup \
  && rm -rf /var/lib/apt/lists/ \
- && rm -rf /var/lib/dpkg/info/
+ && rm -rf /var/lib/dpkg/info/ \
```

Comment:
Can you add a comment explaining why we're doing this?

Reply:
The comment is in the commit message, isn't this enough?

Reply:
OK, added a comment.